### PR TITLE
Add notices to FreeBSD/pfSense docs that they are community-supported

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Users should be aware that these documents aren't actively maintained by the Netdata team.

I also cleaned up the beginning of the pfSense installation procedure. I didn't implement any of the suggestions from #9499, as those are more complex and forward-thinking. Just trying to stop the short-term support issues.

##### Component Name

area/docs
area/packaging

### CODE REVIEW

File: packaging/installer/methods/pfsense.md [hunk 1]
```diff
@@ -30,9 +38,9 @@ pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-yaml-5.2.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.20.0_3.txz
 ```
 
-**Note:** If you receive a `Not Found` error during the last two commands above, you will either need to manually look
-in the [repo folder](http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/) for the latest available package and use its
-URL instead, or you can try manually changing the Netdata version in the URL to the latest version.
+> ❕ If any of the above commands return a `Not Found` error, you need to manually search for the latest package from the
```
Comment (ilyam8) [inline packaging/installer/methods/pfsense.md:41]: probably ⚠️ ?

File: packaging/installer/methods/freebsd.md [hunk 2]
```diff
@@ -19,7 +26,7 @@ pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy
 Download Netdata:
 
 ```sh
-fetch https://github.com/netdata/netdata/releases/download/v1.25.0/netdata-v1.25.0.tar.gz
+fetch https://github.com/netdata/netdata/releases/download/v1.26.0/netdata-v1.26.0.tar.gz
```
Comment (ilyam8) [inline packaging/installer/methods/freebsd.md:29]: Could mention that latest can be checked via:

- browser: https://github.com/netdata/netdata/releases/latest
- cmd: `basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/netdata/netdata/releases/latest)`

I am not sure `curl` is installed on pfsense

```cmd
0 ~ $ basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/netdata/netdata/releases/latest)
v1.26.0
```

Comment (joelhans) [reply packaging/installer/methods/freebsd.md:29]: `curl` is not on pfSense as far as my research goes, maybe for FreeBSD.

Comment (ilyam8) [reply packaging/installer/methods/freebsd.md:29]: Anyway it is a way to get the latest - via browser or cmd (not directly from the pfsense box but from your laptop/other server)

Comment (joelhans) [reply packaging/installer/methods/freebsd.md:29]: Oh, good call!

**Review by ilyam8 — DISMISSED** [review]

File: packaging/installer/methods/pfsense.md [hunk 3]
```diff
@@ -1,47 +1,72 @@
...
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-six-1.15.0.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-cryptography-2.6.1.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-idna-2.8.txz
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-idna-2.10.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-openssl-19.0.0.txz
 pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-pysocks-1.7.1.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-urllib3-1.25.7,1.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-yaml-5.2.txz
-pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/netdata-1.20.0_3.txz
+pkg add http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-urllib3-1.25.10,1.txz
```
Comment (thiagoftsm) [inline packaging/installer/methods/pfsense.md:39]: This URL is wrong, the correct is `http://pkg.freebsd.org/FreeBSD:11:amd64/latest/All/py37-urllib3-1.25.11,1.txz`

Comment (ilyam8) [reply packaging/installer/methods/pfsense.md:39]: It is impossible to keep them up-to-date manually...

The URL was valid 6 days ago

> https://github.com/netdata/netdata/pull/9499#discussion_r513425499

**Review by ilyam8 — APPROVED** [review]

File: packaging/installer/methods/pfsense.md (hunk 3 above)
Comment (joelhans) [reply packaging/installer/methods/pfsense.md]: Yeah, it's going to forever be a moving target until we can figure out a better solution. Let's just get this out the door before it changes again.

**Review by thiagoftsm — APPROVED** [review]
LGTM!

**Pull request merged**